synchronizer/l1_workers_decorator_limit_retries_by_time.go

```diff
@@ -0,0 +1,70 @@
+package synchronizer
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/0xPolygonHermez/zkevm-node/log"
+)
+
+const (
+	cleanUpOlderThan = time.Hour
+)
+
+type controlWorkerFlux struct {
+	time    time.Time
+	retries int
+}
+
+type workerDecoratorLimitRetriesByTime struct {
+	*workers
+	processingRanges    liveBlockRangesGeneric[controlWorkerFlux]
+	minTimeBetweenCalls time.Duration
+}
+
+func newWorkerDecoratorLimitRetriesByTime(workers *workers, minTimeBetweenCalls time.Duration) *workerDecoratorLimitRetriesByTime {
+	return &workerDecoratorLimitRetriesByTime{workers: workers, minTimeBetweenCalls: minTimeBetweenCalls}
+}
+
+func (w *workerDecoratorLimitRetriesByTime) String() string {
+	return fmt.Sprintf("[FILTERED_LRBT Active/%s]", w.minTimeBetweenCalls) + w.workers.String()
+}
+
+func (w *workerDecoratorLimitRetriesByTime) asyncRequestRollupInfoByBlockRange(ctx context.Context, blockRange blockRange) (chan responseRollupInfoByBlockRange, error) {
+	ctrl, err := w.processingRanges.getTagByBlockRange(blockRange)
```

Comment:
I'm still don't know what is the tag. Could you use a more descriptive name?

Reply:
I have add a comment in the definition of the field to clarify the use